### PR TITLE
Update everything

### PR DESCRIPTION
**Pull request by WumboSpasm**
- Rewrote the frontend and most of the backend
- Added Flashpoint 12 support
- Added VRML/X3D support using [X_ITE](https://github.com/create3000/x_ite)
- Added proper backend configuration file
- Added more rigorous server-side checks to test if an entry is supported
- Revamped frontend UI to fit better with entry browser
- Improved FAQ

### CODE REVIEW

File: static/main.js [hunk 1]
```diff
@@ -1,158 +1,155 @@
-// Global variable that will contain the entry's UUID
-let uuid = '';
+const oooo = 'http://127.0.0.1:8985';
```
Comment (n0samu) [inline static/main.js:1]: Looks leftover from testing?

Comment (WumboSpasm) [reply static/main.js]: Fixed

**Comment by Dri0m** [comment]
image

- it was important that this mentioned that it's compatibility rating, not a score of the game
- it should be `N/A` or anything else when there are no votes

File: static/index.html [hunk 2]
```diff
@@ -1,133 +1,39 @@
...
+        <script src="main.js" defer></script>
+    </head>
+    <body>
+        <div class="content">
+            <div class="header"></div>
+            <div class="player"></div>
+            <div class="footer">
+                <div class="navigation">
+                    <div class="general">
+                        <a href="faq" target="_blank">[FAQ]</a>
```
Comment (Dri0m) [inline static/index.html:16]: missing link?

**Pull request merged**